packages/lambda-tiler/src/util/response.ts

```diff
@@ -3,3 +3,4 @@ import { LambdaHttpResponse } from '@linzjs/lambda';
 export const NotFound = (): LambdaHttpResponse => new LambdaHttpResponse(404, 'Not Found');
 export const NotModified = (): LambdaHttpResponse => new LambdaHttpResponse(304, 'Not modified');
 export const NoContent = (): LambdaHttpResponse => new LambdaHttpResponse(204, 'No Content');
+export const OkResponse = (): LambdaHttpResponse => new LambdaHttpResponse(200, 'ok');
```

Comment:
You should respond with some sort of body, maybe copy the Error body example where it puts the ids and a message as a JSON response.

You should be able to do something like

`LambdaHttpResponse.ok().json({ id: req.id, correladtionId: req.correlationId, message: 'ok' })`

Reply:
Fixed. Also, I have updated the name as `OkPostResponse`;

Reply:
Does it need to be specific to POST? could just be a `OkResponse` as you could use it as a PUT too?

Reply:
I think you are correct. Just reverted it back. Should be ready now.